src/crawlee/_autoscaling/system_status.py

```diff
@@ -10,7 +10,6 @@
 
 from crawlee._autoscaling.types import LoadRatioInfo, Snapshot, SystemInfo
 from crawlee._utils.docs import docs_group
-from crawlee._utils.math import compute_weighted_avg
```

Comment:
Could you delete the `compute_weighted_avg` definition and its tests? It seems it is not used anymore.